diff --git a/node_modules/react-native-reanimated/ios/RNReanimated.xcodeproj/xcuserdata/mdk.xcuserdatad/xcschemes/xcschememanagement.plist b/node_modules/react-native-reanimated/ios/RNReanimated.xcodeproj/xcuserdata/mdk.xcuserdatad/xcschemes/xcschememanagement.plist
index 2e11062..98fd95a 100644
--- a/node_modules/react-native-reanimated/ios/RNReanimated.xcodeproj/xcuserdata/mdk.xcuserdatad/xcschemes/xcschememanagement.plist
+++ b/node_modules/react-native-reanimated/ios/RNReanimated.xcodeproj/xcuserdata/mdk.xcuserdatad/xcschemes/xcschememanagement.plist
@@ -12,7 +12,7 @@
 		<key>RNReanimated.xcscheme_^#shared#^_</key>
 		<dict>
 			<key>orderHint</key>
-			<integer>3</integer>
+			<integer>4</integer>
 		</dict>
 	</dict>
 </dict>
diff --git a/node_modules/react-native-reanimated/react-native-reanimated.d.ts b/node_modules/react-native-reanimated/react-native-reanimated.d.ts
index 7a30d62..f21493c 100644
--- a/node_modules/react-native-reanimated/react-native-reanimated.d.ts
+++ b/node_modules/react-native-reanimated/react-native-reanimated.d.ts
@@ -141,8 +141,10 @@ declare module 'react-native-reanimated' {
 
     export const SpringUtils: SpringUtils
 
+    type AnimatedTransform = { [P in keyof TransformStyle["transform"]]: Animated.Adaptable<TransformStyle["transform"][P]> };
+
     type AnimateStyle<S extends object> = {
-      [K in keyof S]: S[K] extends ReadonlyArray<any>
+      [K in keyof S]: K extends 'transform' ? AnimatedTransform : (S[K] extends ReadonlyArray<any>
         ? ReadonlyArray<AnimateStyle<S[K][0]>>
         : S[K] extends object
           ? AnimateStyle<S[K]>
@@ -151,7 +153,7 @@ declare module 'react-native-reanimated' {
               | AnimatedNode<
                   // allow `number` where `string` normally is to support colors
                   S[K] extends string ? S[K] | number : S[K]
-                >
+                >)
     };
 
     type AnimateProps<
